pyproject.toml

```diff
@@ -1,20 +1,53 @@
-[tool.black]
-line-length = 79
-exclude = '''
-(
-  /(  # generated files
-    .tox|
-    venv|
-    venv.*|
-    .venv.*|
-    target.*|
-    .*/build/lib/.*|
-    exporter/opentelemetry-exporter-zipkin-proto-http/src/opentelemetry/exporter/zipkin/proto/http/v2/gen|
-    opentelemetry-proto/src/opentelemetry/proto/.*/.*|
-    scripts
-  )/
-)
-'''
 [tool.pytest.ini_options]
 addopts = "-rs -v"
 log_cli = true
+
+[tool.ruff]
+# Minimum supported python version
+target-version = "py38"
+
+line-length = 79
+extend-exclude = [
+  # Ignore generated files
+  "exporter/opentelemetry-exporter-zipkin-proto-http/src/opentelemetry/exporter/zipkin/proto/http/v2/gen",
+  "*_pb2*.py*",
+]
+
+[tool.ruff.lint]
```

Comment:
I think this block should be on the other PR

Reply:
Done